app/dashboard/views.py

```diff
@@ -6494,6 +6496,62 @@ def verify_user_poap(request, handle):
     )
 
 
+@login_required
+@require_POST
+def verify_user_poh(request, handle):
+    is_logged_in_user = request.user.is_authenticated and request.user.username.lower() == handle.lower()
+    if not is_logged_in_user:
+        return JsonResponse({
+            'ok': False,
+            'msg': f'Request must be for the logged in user'
+        })
```

Comment:
^ this is not needed as `login_required` is already used

Reply:
So is it ok to remove `handle` URL parameter and use `request.user.username`?
Otherwise, the check is required to make sure the user entered her/his handle only.